tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -134,6 +137,7 @@ private FileSystemMaster createFsMasterFromJournal() throws IOException {
    * Reproduce the journal and check if the state is correct.
    */
   private void reproduceAndCheckState(int successFiles) throws Exception {
+    Assert.assertNotEquals(successFiles, 0);
```

Comment:
If successFiles are zero the listStatus below will throw an error with FileNotFounException.